houdini/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN --mount=type=cache,target=/var/cache/apk \
     --mount=type=bind,from=imagemagick,source=/home/builder/packages/x86_64,target=/packages \
     --mount=type=bind,from=imagemagick,source=/etc/apk/keys,target=/etc/apk/keys \
     --mount=type=cache,target=/root/.composer/cache \
-    apk add /packages/imagemagick-*.apk && \
+    apk add --allow-untrusted /packages/imagemagick-*.apk && \
```

Comment:
This line stands out to me, but honestly, I don't know what "allow-untrusted" opens us up to.   So, just noting concern, but not sure if it's warranted.

Reply:
I explain in one of the commits - basically, the .apk package for imagemagick is custom-built in the `imagemagick` image.  The `houdini` image mounts it and installs it.  In theory, the `imagemagick` image creates keys that are then mounted by `houdini` at build-time, which is supposed to make `apk` trust the custom-built-and-signed imagemagick pakage.  For some reason, this trust isn't happening, and `apk` is complaining that imagemagick isn't signed by a trusted signature.

I think the trust mechanism must have become stricter somehow, but I ran out of time and patience.  Since it's a matter of one of our images trusting a package that another one of our images just built, I figured the `--allow-untrusted` is OK in this case.

Reply:
APK format packages have a gpg signature.  `--allow-untrusted` means that it will no longer care if the gpg matches or even exists. The risk is minimal because we build the package ourselves.